pallets/subtensor/src/lib.rs

```diff
@@ -1959,6 +1969,18 @@ pub mod pallet {
         ) -> DispatchResult {
             Self::do_sudo_set_adjustment_alpha(origin, netuid, adjustment_alpha)
         }
+
+        #[pallet::call_index(59)]
+        #[pallet::weight((Weight::from_ref_time(14_000_000)
```

Comment:
Was this benchmarked? If not, will add it to the list of features that need benchmarking before we push latest chain upgrade.

Reply:
It was not

Reply:
> Was this benchmarked? If not, will add it to the list of features that need benchmarking before we push latest chain upgrade.

No it wasn't benchmarked, but copied the time of another u16 hyperparam.